### ISSUE TITLE
Editorial: More correctly link in §0.1 on WCAG 2.1

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#608]
https://w3c.github.io/wcag/21/guidelines/#background-on-wcag-2

current:
We encourage authors to refer to our supplemental guidance on [improving inclusion for people with disabilities, including learning and cognitive disabilities, people with low-vision, and more](https://www.w3.org/WAI/standards-guidelines/wcag/#supplement).

proposal:
We encourage authors to refer to our [supplemental guidance](https://www.w3.org/WAI/standards-guidelines/wcag/#supplement) on improving inclusion for people with disabilities, including learning and cognitive disabilities, people with low-vision, and more.

**Comment by awkawk** [comment]
Proposed response: Thank you for the comment. The Working Group believes that both are acceptable so this does not meet the threshold for an error in the spec. We can reconsider in the next normative update.

**Comment by Rafalchar** [comment]
I would say that:
<br>
We encourage authors to refer to our <a href="https://www.w3.org/WAI/standards-guidelines/wcag/#supplement">supplemental guidance on improving inclusion for people with disabilities, including learning and cognitive disabilities, people with low-vision, and more.</a>
<br>
would be even better. It would be friendly for screen readers users.
